css/src/components/breadcrumbs.scss

```diff
@@ -0,0 +1,30 @@
+$breadcrumb-font-size: $font-size-8 !default;
+$breadcrumb-spacing-horizontal: 0.25em !default;
+$breadcrumb-spacing-vertical: 0.375em !default;
+$breadcrumb-separator: '\00200B/' !default;
+
+.breadcrumbs {
+	font-size: $breadcrumb-font-size;
+	list-style: none;
+
+	.breadcrumb-item {
+		display: inline;
+
+		@include desktop {
+			display: inline-block;
+			padding-block: $breadcrumb-spacing-vertical;
+		}
+
+		&::after,
+		&::before {
+			display: inline-block;
+			margin-inline: $breadcrumb-spacing-horizontal;
+			color: $text-subtle;
+		}
+
+		&:not(.breadcrumb-slashless)::after {
```

Comment:
We could add the .breadcrumb-slashless modifier later, as-needed. In the new design every crumb is terminated with a slash.

Reply:
After thinking about it for a while I still think it's good to add it now since Atlas is meant to be used by anyone. And that's a high chance people would want to include the last crumb, even though in our project it's not the case.

Reply:
We need to be careful here- Atlas is *not* built such that it serves everyone's project needs. The purpose of Atlas is to serve the Docs platform, developed in the open to exhibit our patterns and practices with respect to modern css and accessibility.
That said, I do not see a major issue in including such a small extension like this. 

To clean this up, let's combine with the preceding selector and update the modifier class name to match our standards.

```scss
&::after:not(.breadcrumb-item-slashless) {
	display: inline-block;
	margin-inline: $breadcrumb-spacing-horizontal;
	color: $text-subtle;
	content: $breadcrumb-separator;
}
```

Reply:
Updated!